esp/src/eclwatch/LFDetailsWidget.js

```diff
@@ -340,7 +363,10 @@ define([
                 //  Force Icon to Show (I suspect its not working due to Circular Reference Loading)
                 this.queriesWidget.set("iconClass", "dijitInline dijitIcon dijitTabButtonIcon iconFind");
             } else if (name === "DFUFilePartsOnClusters") {
-            	// Currently only checking first cluster may add loop through clusters and add a tab at a later date
+            	if (!newValue.DFUFilePartsOnCluster[0].CanReplicate || newValue.DFUFilePartsOnCluster[0].CanReplicate === false || newValue.DFUFilePartsOnCluster[0].Replicate === true) {
```

Comment:
CanReplicate can be missing from response and it also can be there and equal false.